main.py

```
@@ -18,108 +21,128 @@ def print_section(title):
-    loader = DataLoader()
-    analyzer = ReviewAnalyzer()
 
-    # Step 1: Load data
+def load_reviews(loader):
+    """Load reviews and filter negative ones."""
     print_section("Step 1: Loading Data")
     try:
         df = loader.load_reviews()
-    except Exception as e:
-        print(f"\n❌ Error loading data: {e}")
+    except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)